### ISSUE TITLE
Error configuring with CMake on Windows

### ISSUE THREAD

**Issue by Bleach665** [opencv/dldt#386]
Windows 10, VS 2017, CMake gui 3.13.4, CUDA 10.1.
UPD. I use old CPU which not support AVX, only SSE4.
DLDT was cloned 
`git clone https://github.com/opencv/dldt.git --recurse-submodules`
and after configured via CMake gui.
Error received:

````
Selecting Windows SDK version 10.0.17763.0 to target Windows 10.0.14393.
CMAKE_BUILD_TYPE: Release
Detected 64 bit architecture
Inference Engine enabled features: 

    CI_BUILD_NUMBER: custom_2020_b2140c083a068a63591e8c2e9b5f6b240790519d
    ENABLE_NGRAPH = ON
    ENABLE_INFERENCE_ENGINE = ON
    ENABLE_MKL_DNN = ON
    ENABLE_CLDNN = ON
    ENABLE_LTO = OFF
    OS_FOLDER = OFF
    TREAT_WARNING_AS_ERROR = OFF
    ENABLE_SANITIZER = OFF
    ENABLE_THREAD_SANITIZER = OFF

PROJECT ............................... OpenVINO
CMAKE_BINARY_DIR ...................... E:/Lib_prebuild/DLDT/prebuild_x64/inference-engine
OpenVINO_MAIN_SOURCE_DIR .............. E:/Lib_prebuild/DLDT/source
IE_MAIN_SOURCE_DIR .............. E:/Lib_prebuild/DLDT/source/inference-engine
CMAKE_GENERATOR ....................... Visual Studio 15 2017 Win64
CMAKE_C_COMPILER_ID ................... MSVC
CMAKE_BUILD_TYPE ...................... Release
Current hash b0bb801c91f091a125a6aeb38c51f10fc49f3425
NGRAPH_VERSION 0.27.1-rc.0+b0bb801
NGRAPH_VERSION_SHORT 0.27.1-rc.0
NGRAPH_WHEEL_VERSION 0.27.1rc0
NGRAPH_API_VERSION 0.27
NGRAPH_FORWARD_CMAKE_ARGS -DCMAKE_C_COMPILER=C:/Program Files (x86)/Microsoft Visual Studio/2017/Enterprise/VC/Tools/MSVC/14.16.27023/bin/Hostx86/x64/cl.exe;-DCMAKE_CXX_COMPILER=C:/Program Files (x86)/Microsoft Visual Studio/2017/Enterprise/VC/Tools/MSVC/14.16.27023/bin/Hostx86/x64/cl.exe;-DCMAKE_CXX_STANDARD:STRING=11;-DCMAKE_CXX_STANDARD_REQUIRED:BOOL=ON;-DCMAKE_CXX_EXTENSIONS:BOOL=OFF;-DCMAKE_EXPORT_COMPILE_COMMANDS:BOOL=ON;-DCMAKE_POSITION_INDEPENDENT_CODE:BOOL=ON
NGRAPH_CXX_STANDARD:                  11
NGRAPH_UNIT_TEST_ENABLE:              ON
NGRAPH_DOC_BUILD_ENABLE:              OFF
NGRAPH_TOOLS_ENABLE:                  OFF
NGRAPH_CPU_ENABLE:                    OFF
NGRAPH_USE_LEGACY_MKLDNN:             OFF
NGRAPH_MLIR_ENABLE:                   OFF
NGRAPH_INTERPRETER_ENABLE:            OFF
NGRAPH_NOP_ENABLE:                    OFF
NGRAPH_GENERIC_CPU_ENABLE:            OFF
NGRAPH_DEBUG_ENABLE:                  OFF
NGRAPH_DEPRECATED_ENABLE:             OFF
NGRAPH_ONNX_IMPORT_ENABLE:            OFF
NGRAPH_DEX_ONLY:                      OFF
NGRAPH_ENABLE_CPU_CONV_AUTO:          OFF
NGRAPH_CODE_COVERAGE_ENABLE:          OFF
NGRAPH_LIB_VERSIONING_ENABLE:         OFF
NGRAPH_PYTHON_BUILD_ENABLE:           OFF
NGRAPH_USE_PREBUILT_LLVM:             OFF
NGRAPH_PLAIDML_ENABLE:                OFF
NGRAPH_DISTRIBUTED_ENABLE:            OFF
NGRAPH_JSON_ENABLE:                   OFF
NGRAPH_STATIC_LIB_ENABLE:             OFF
NGRAPH_INTERPRETER_STATIC_LIB_ENABLE: OFF
NGRAPH_CPU_STATIC_LIB_ENABLE:         OFF
NGRAPH_PLAIDML_STATIC_LIB_ENABLE:     OFF
NGRAPH_DYNAMIC_COMPONENTS_ENABLE:     OFF
NGRAPH_NATIVE_ARCH_ENABLE:            OFF
Installation directory: C:/Program Files/OpenVINO
Compile Flags: /DWIN32 /D_WINDOWS /W0 /GR /EHsc  /sdl -D_CRT_SECURE_NO_WARNINGS -D_SCL_SECURE_NO_WARNINGS /EHsc /Gy /bigobj /MP /std:c++14 /wd4251 /wd4275 /wd4308 /wd4146 -DEIGEN_MPL2_ONLY -DTBB_USE_THREADING_TOOLS /EHsc /MP /bigobj /wd4251 /wd4275 -DNOMINMAX -D_CRT_SECURE_NO_WARNINGS
Shared Link Flags: /machine:x64
CMAKE_CXX_FLAGS_RELEASE /MD /O2 /Ob2 /DNDEBUG
CMAKE_CXX_FLAGS_DEBUG /MDd /Zi /Ob0 /Od /RTC1 /Z7
NGRAPH_FLUID_ENABLE:                  ON
NGRAPH_FLUID_TEST_ENABLE:             OFF
tools disabled
unit tests enabled
NGRAPH_UNIT_TEST_OPENVINO_ENABLE = TRUE
NGRAPH_TESTS: CPU enabled
NGRAPH_TESTS: GPU enabled
Adding unit test for backend CPU
Adding unit test for backend GPU
INTEL_VTUNE_DIR is not defined
Could NOT find INTEL_ITT (missing: Located_ITT_INCLUDE_DIRS Located_ITT_LIBS) 
Inference Engine enabled features: 

    CI_BUILD_NUMBER: custom_2020_b2140c083a068a63591e8c2e9b5f6b240790519d
    ENABLE_NGRAPH = ON
    ENABLE_INFERENCE_ENGINE = ON
    ENABLE_MKL_DNN = ON
    ENABLE_CLDNN = ON
    ENABLE_LTO = OFF
    OS_FOLDER = OFF
    TREAT_WARNING_AS_ERROR = OFF
    ENABLE_SANITIZER = OFF
    ENABLE_THREAD_SANITIZER = OFF
    ENABLE_GNA = ON
    ENABLE_CLDNN_TESTS = OFF
    ENABLE_PROFILING_ITT = ON
    ENABLE_PROFILING_RAW = OFF
    GEMM = JIT
    THREADING = TBB
    ENABLE_VPU = ON
    ENABLE_MYRIAD = ON
    ENABLE_MYRIAD_NO_BOOT = OFF
    ENABLE_TESTS = OFF
    ENABLE_GAPI_TESTS = OFF
    GAPI_TEST_PERF = OFF
    ENABLE_MYRIAD_MVNC_TESTS = OFF
    ENABLE_BEH_TESTS = OFF
    ENABLE_FUNCTIONAL_TESTS = OFF
    ENABLE_SAMPLES = ON
    ENABLE_FUZZING = OFF
    COVERAGE = OFF
    VERBOSE_BUILD = OFF
    ENABLE_UNSAFE_LOCATIONS = OFF
    ENABLE_ALTERNATIVE_TEMP = ON
    ENABLE_DUMP = OFF
    ENABLE_OPENCV = ON
    ENABLE_DEBUG_SYMBOLS = OFF
    ENABLE_PYTHON = OFF
    ENABLE_CPP_CCT = OFF
    ENABLE_UNICODE_PATH_SUPPORT = ON
    ENABLE_IR_READER = ON
    ENABLE_C = ON
    ENABLE_CPPLINT = OFF
    ENABLE_CPPLINT_REPORT = OFF
    ENABLE_CLANG_FORMAT = OFF
    ENABLE_CPPCHECK = OFF
    GNA_LIBRARY_VERSION = GNA1_1401

DL_SDK_TEMP envionment not set
[clDNN] $(Configuration) configuration is not supported for prebuilt dependencies. Release will be used
[clDNN] ======================== clDNN Project =======================
[clDNN] Version:          1.4.24.0
[clDNN]
[clDNN] Build type:       Release (for single-configuration generators)
[clDNN] Av. build types:  Debug;Release;RelWithDebInfo (for multi-configuration generators)
[clDNN]
[clDNN] Output bin directory:
[clDNN]  - "E:/Lib_prebuild/DLDT/source/bin/intel64"
[clDNN] Output lib directory:
[clDNN]  - "E:/Lib_prebuild/DLDT/source/bin/intel64"
[clDNN] Architecture:
[clDNN]  - target: Windows64    (detected: Windows64)
[clDNN]
[clDNN]
[clDNN] Advanced:
[clDNN]  - ICD version used to build:         6.3
[clDNN]
[clDNN]  - Include/Build cldnn core:          ON
[clDNN]  - Include/Build kernel selector:     ON
[clDNN]  - Include/Build tests:               OFF
[clDNN]  - Include/Build core internal tests: OFF
[clDNN]  - Include/Build tutorial:            OFF
[clDNN]
[clDNN]  - Run tests:                     OFF
[clDNN]  - Run core internal tests:       OFF
[clDNN]
[clDNN]  - Use static C++ Runtime:        OFF
[clDNN]  - Allow unsafe size opts:        ON
[clDNN]  - CMake debug trace:             OFF
[clDNN]
[clDNN]
[clDNN] ICD:
[clDNN]  - Root:            E:/Lib_prebuild/DLDT/source/inference-engine/thirdparty/clDNN/common/intel_ocl_icd/6.3
[clDNN]     + Headers:      E:/Lib_prebuild/DLDT/source/inference-engine/thirdparty/clDNN/common/intel_ocl_icd/6.3/windows/include
[clDNN]     + Static libs:  E:/Lib_prebuild/DLDT/source/inference-engine/thirdparty/clDNN/common/intel_ocl_icd/6.3/windows/Release/lib/x64
[clDNN]     + Shared libs:  E:/Lib_prebuild/DLDT/source/inference-engine/thirdparty/clDNN/common/intel_ocl_icd/6.3/windows/Release/bin/x64
[clDNN]     + Libs to link: E:/Lib_prebuild/DLDT/source/inference-engine/thirdparty/clDNN/common/intel_ocl_icd/6.3/windows/Release/lib/x64/OpenCL.lib
[clDNN] =============================================================================
[clDNN] Selected capabilities: public
CMake Error at inference-engine/thirdparty/pugixml/CMakeLists.txt:80 (list):
  list sub-command REMOVE_ITEM requires two or more arguments.


libGNA::API INTERFACE_LIBRARY : E:/Lib_prebuild/DLDT/source/inference-engine/temp/gna_01.00.00.1401/include
libGNA::KERNEL SHARED_LIBRARY : E:/Lib_prebuild/DLDT/source/inference-engine/temp/gna_01.00.00.1401/win64/x64/gna.lib
The name ngraph::ngraph is an ALIAS for ngraph. It will be exported to the InferenceEngineDeveloperPackage with the original name.
Configuring incomplete, errors occurred!
See also "E:/Lib_prebuild/DLDT/prebuild_x64/inference-engine/CMakeFiles/CMakeOutput.log".
See also "E:/Lib_prebuild/DLDT/prebuild_x64/inference-engine/CMakeFiles/CMakeError.log".
````

[CMakeFiles.zip](https://github.com/opencv/dldt/files/4204610/CMakeFiles.zip)

**Comment by jgespino** [comment]
Hi @Bleach665 

I did a quick test on my Windows 10 with CMake 3.14.4, Microsoft Visual Studio 2019 and Python 3.7.3.  I did not see the same issue you are seeing, I used the [build instructions](https://github.com/opencv/dldt/blob/2020/build-instruction.md#build-on-windows-systems) for Windows.

`git clone https://github.com/opencv/dldt`
`cd dldt`
`git submodule update --init --recursive`
`mkdir build`
`cmake -G "Visual Studio 16 2019" -A x64 -DCMAKE_BUILD_TYPE=Release ..`

Which processor are you using?

Regards,
Jesus
